front/src/components/App/MaintenanceArea/AddAuction.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const AddAuction = (auctions: string[], newAuction: string) => {
```

Comment:
to zwykła funkcja, a nie funkcyjny komponent. Zamiast tworzyć dla każdej operacji CRUDa osobny plik można zrobić np jeden plik utils.ts i tam zebrać wszystkie dodatkowe funkcje.
Ponadto nazwy funkcji należy zaczynać małymi literami